plugin/storage/es/options.go

```diff
@@ -112,6 +108,14 @@ func NewOptions(primaryNamespace string, otherNamespaces ...string) *Options {
 	return options
 }
 
+func (config *namespaceConfig) GetTLSFlagsConfig() *tlscfg.ClientFlagsConfig {
```

Comment:
- does it need to be public?
- it can return config by value, we're not using pointers anywhere